### ISSUE TITLE
How big a vmware environment can this handle?

### ISSUE THREAD

**Issue by trippinnik** [pryorda/vmware_exporter#315]
Having trouble getting the metrics page to successfully complete. It looks as though prometheus scrape requests are triggering it to start pulling the metrics again

I've tried using VSPHERE_SPECS_SIZE with various sizes but that has not helped.

**Comment by daverstam** [comment]
I'm running this on a platform with approx 200+ hosts and ~ 2000vm's and it usually takes around 30s to scrape. This is the config I'm using:
 
`default:
    vsphere_host: "***"
    vsphere_user: "***"
    vsphere_password: "***"
    ignore_ssl: True
    specs_size: 5000
    fetch_custom_attributes: False
    fetch_tags: False
    fetch_alarms: True
    collect_only:
        vms: True
        vmguests: False
        datastores: True
        hosts: True
        snapshots: False`

**Comment by dcorneli** [comment]
Yes I had to increase the scrape interval by quite a bit otherwise the scraping would take so long that the next scrape would start before the previous one was finished.
This is what I'm using:

scrape_timeout: 60s
scrape_interval: 180s
